devtools/lightning-inspect.es5.js

```diff
@@ -762,23 +762,12 @@
           this.__renderOffscreen = v;
         }
       }
-    }); // defining the isfocused property and attribute
-
-    ElementCore.prototype._isfocused = false;
-    Object.defineProperty(ElementCore.prototype, 'isfocused', {
-      // We are working on the core!
-      get: function get() {
-        console.log('Inspector geeting is focused');
-        return this._isfocused;
-      },
-      set: function set(v) {
-        console.log('Inspector setting is focused');
-        this._isfocused = v; // Put the attribute 'isfocused' as the current value
-
-        val(this, 'isfocused', v, false);
-      }
     });
 
+    Element.prototype.updateDebugFocusState = function () {
+      val(this, 'hasfinalfocus', this.hasFinalFocus(), false);
```

Comment:
Leaning on the existing hasFinalFocus may minimise the chance of inconsistent representation of focus state.